BepressImportDom.inc.php

```diff
@@ -618,9 +623,26 @@ function _handlePDFGalleyNode() {
 	}
 
 	function _getArticleTitle() {
-		$titleNode = $this->_articleNode->getChildByName('title');
-		$title = $titleNode->getValue();
-		$this->_articleTitle = html_entity_decode(trim($title), ENT_HTML5);
+		$titleLocalizedArray = $this->_getLocalizedElements($this->_articleNode, 'title', 'titles');
+
+		// Check if we have a title for primary locale, if not assign first title element to primary locale
+		$containsPrimaryLocale = false;
+		foreach ($titleLocalizedArray as $locale => $titleList) {
+			if ($locale === $this->_primaryLocale) {
+				$containsPrimaryLocale = true;
+				break;
+			}
+		}
+
+		if (!$containsPrimaryLocale) {
+			$titleLocalizedArray[$this->_primaryLocale] = $titleLocalizedArray[0];
+		}
+		$this->_articleTitleLocalizedArray = $titleLocalizedArray;
+		$this->_articleTitle = $this->_articleTitleLocalizedArray[$this->_primaryLocale][0];
+
+//		$titleNode = $this->_articleNode->getChildByName('title');
```

Comment:
@ewhanson, looks like some stray commented-out code here.

Reply:
Thanks. Taken care of.